src/ops/ops.go

```diff
@@ -463,3 +465,44 @@ func (o *ops) CreateOpenshiftSshManifest(filePath, tmpl, sshPubKeyPath string) e
 	}
 	return nil
 }
+
+func (o *ops) GetMustGatherLogs(workDir string, kubeconfigPath string) (string, error) {
+	//invoke oc adm must-gather command in the working directory
+	o.log.Infof("collecting must-gather logs into %s", workDir)
+	command := fmt.Sprintf("cd %s && oc --kubeconfig=%s adm must-gather", workDir, kubeconfigPath)
```

Comment:
can we have must-gather as consts?

Reply:
it's less clear as const. this is self explanatory to the reader and it's actually being used just once (here). all the other places are comments or part of directory names

Reply:
but we have 7 times must-gather in different places